### DISCUSSION TITLE
Beware: the "local only" feature will permanently delete your notes !!!!

### DISCUSSION THREAD

**Discussion by Lorenzo795** [streetwriters/notesnook#2556] [General]
Just to let you guys know that if you make any of your notes "local only", the next time you relaunch the app those notes will be gone.

I went through the support channel here and the answer back was "that's weird".

I lost weeks of work with those notes that are gone, and it's really disappointing because there's no way to recover all that work.

So, just a heads up to all you guys.

**Comment by thecodrr** [comment]
I don't see the point of this. You already opened an issue for this #2502.
